src/shaka_scheme/system/gc/GCData.cpp

```diff
@@ -4,39 +4,40 @@
 #include "shaka_scheme/system/gc/GCData.hpp"
 
 namespace shaka {
+    namespace gc {
 
-    GCData::GCData(const Data& data): data(data),
-            marked(false), next(nullptr) {}
+        GCData::GCData(const Data &data) : marked(false),
```

Comment:
Please fix the formatting here so that it matches [here](https://github.com/uhmanoa-transpiler-project/shaka-scheme/blob/master/src/shaka_scheme/system/base/Vector.cpp).

Reply:
Ok fixed!